### PR TITLE
fix #4, add terminationDrationDuration config

### LINKED ISSUE #4: add terminationDrainDuration to avoid unpaid requests to fail
- [ ] I have searched the [issues](https://github.com/cectc/dbpack/issues) of this repository and believe that this is not a duplicate.

**Describe the bug**
In sidecar model, Kubernetes will send SIGTERM signals to all containers at the same time before destroying the pod, so dbpack container will stop at the same time as business containers, this may cause unpaid requests to fail.

I think DBPack should waiting for a length of time （terminationDrainDuration） before exiting can solve this problem. Just like Istio-Proxy does.

related code（cmd/cmd.go#L177）
`
			dbpack.Start()

			ctx, cancel := context.WithCancel(context.Background())
			c := make(chan os.Signal, 2)
			signal.Notify(c, os.Interrupt, syscall.SIGTERM)
			go func() {
				<-c
				cancel()
				<-c
				os.Exit(1) // second signal. Exit directly.
			}()

			<-ctx.Done()
`

### PR DESCRIPTION
**Pull request by gorexlv**
ref: https://github.com/cectc/dbpack/issues/<issueID>

### Ⅰ. Describe what this PR did

fix #4 

### Ⅱ. Does this pull request fix one issue?

#4

### CODE REVIEW

**Comment by gorexlv** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by dk-lockdown — APPROVED** [review]
lgtm

**Review by bohehe — APPROVED** [review]
lgtm

File: cmd/cmd.go [hunk 1]
```diff
@@ -117,41 +118,42 @@ var (
...
 					listener, err := listener.NewMysqlListener(listenerConf)
 					if err != nil {
-						panic(err)
+						log.Fatalf("create mysql listener failed %v", err)
 					}
 					dbListener := listener.(proto.DBListener)
 					executor := executors[listenerConf.Executor]
 					if executor == nil {
-						panic(errors.Errorf("executor: %s is not exists for mysql listener", listenerConf.Executor))
+						log.Fatalf("executor: %s is not exists for mysql listener", listenerConf.Executor)
```
Comment (zackzhangkai) [inline cmd/cmd.go:126]: If a fatal error accounted, maybe we should stop progress going on.

Comment (dk-lockdown) [reply cmd/cmd.go:126]: this func, after print log, will call os.Exit

**Comment by fatelei** [comment]
LGTM

File: cmd/cmd.go (hunk 1 above)
Comment (zackzhangkai) [reply cmd/cmd.go:126]: Got it.

**Comment by zackzhangkai** [comment]
lgtm

**Pull request merged**